DQM/Integration/python/clients/beam_dqm_sourceclient-live_cfg.py

```diff
@@ -385,6 +385,12 @@
             onlyAppendUpdatePolicy = cms.untracked.bool(True)
         ))
     )
+    try:
+        process.OnlineDBOutputService.frontierKey = cms.untracked.string(options.runUniqueKey)
+        print("Configured frontierKey", options.runUniqueKey)
+    except:
+        pass
```

Comment:
@smorovic @cms-sw/dqm-l2 @cms-sw/alca-l2 @cms-sw/db-l2 isn't it better to add a message also in the "except" branch ?

Reply:
@silviodonato
with all possible input sources used in this and similar configuration files now having the runUniqueKey VarArgs option, we could also just remove try/except and set frontierKey within process.OnlineDBOutputService definition. Does this seem ok for @cms-sw/dqm-l2 and others?

Reply:
Yes, fine from our side. Thanks

Reply:
Pushed the change. If it looks ok, I'll also add it to 11_3_X backport.